docs/services/gcp-services-guide.md

```diff
@@ -0,0 +1,536 @@
+  namespace: app-project1-dev
+classRef:
+  kind: CloudsqlInstanceClass
+  apiVersion: database.gcp.crossplane.io/v1alpha2
+  name: standard-cloudsql
+  namespace: $INFRA_NAMESPACE
+EOF
+
+kubectl apply -f mysql-standard.yaml
+```
+
+* You can verify creation with the following command and output:
+
+*Command*
+```bash
+kubectl get mysqlinstanceclasses -n app-project1-dev
+```
+*Output*
+```bash
+NAME             AGE
+mysql-standard   27s
+```
+
+Once again, you are free to create more `MySQLInstanceClass` instances in this
+namespace to define more classes of service. For instance, if you created
+`mysql-gcp-large` above, you may want to create a `MySQLInstanceClass` named
+`mysql-large` that references it. You may also choose to create MySQL resource
+classes for other non-GCP providers, and reference them for a class of service
+in the `app-project1-dev` namespace.
+
+You may specify *one* instance of a portable class kind as *default* in each
+namespace. This means that the portable resource class instance will be applied
+to claims that do not directly reference a portable class. If we wanted to make
+our `mysql-standard` instance the default `MySQLInstanceClass` for namespace
+`app-project1-dev`, we could do so by adding a label:
+
+```bash
+kubectl -n app-project1-dev label mysqlinstanceclass mysql-standard default="true"
+```
+
+#### Resource Claims
+Resource claims are used to create external resources by referencing a class of
+service in the claim's namespace. When a claim is created, Crossplane uses the
+referenced portable class to find a cloud-specific resource class to use as the
+configuration for the external resource. We need a to create a claim to
+provision the MySQL database we will use in GCP.
+
+* Define a `MySQLInstance` claim in `mysql-claim.yaml`:
+> You can omit `spec.classRef` if you want to use the default MySQLInstanceClass.
+```bash
+cat > mysql-claim.yaml <<EOF
+---
+apiVersion: database.crossplane.io/v1alpha1
+kind: MySQLInstance
+metadata:
+  name: mysql-claim
+  namespace: app-project1-dev
+spec:
+  classRef:
+    name: mysql-standard
+  engineVersion: "5.7"
+  # A secret is exported by providing the secret name
+  # to export it under. This is the name of the secret
+  # in the crossplane cluster, and it's scoped to this claim's namespace.
+  writeConnectionSecretToRef:
+    name: wordpressmysql
+EOF
+
+kubectl apply -f mysql-claim.yaml
+```
+
+What we are looking for is for `STATUS` value to become `Bound` which indicates
+the managed resource was successfully provisioned and is ready for consumption.
+You can see when claim is bound using the following:
+
+*Command*
+```bash
+kubectl get mysqlinstances -n app-project1-dev
+```
+
+*Output*
+```bash
+NAME          STATUS   CLASS            VERSION   AGE
+mysql-claim   Bound    mysql-standard   5.6       11m
+```
+
+If the `STATUS` is blank, we are still waiting for the claim to become bound.
+You can observe resource creation progression using the following:
```

Comment:
Can we can also use `--wait` to wait for the status we want? If so, it may make sense to show the user how to do that.